src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -332,6 +338,9 @@ namespace ts.server.typingsInstaller {
                     if (this.log.isEnabled()) {
                         this.log.writeLine(`Installed typings ${JSON.stringify(scopedTypings)}`);
                     }
+
+                    this.writePackageVersion(cachePath);

```

Comment:
That being said, I should probably only do the actual file write if the version did change.